pkg/api/account_handlers.go

```diff
@@ -155,3 +155,23 @@ func (h Handler) GetAccountTransactions(ctx context.Context, params oas.GetAccou
 	}
 	return &result, nil
 }
+
+func (h Handler) GetSearchAccounts(ctx context.Context, params oas.GetSearchAccountsParams) (res oas.GetSearchAccountsRes, err error) {
+	if len(params.Name) < 3 {
```

Comment:
you can add it directly to swagger 
minLength: 3
maxLength: 15
and is should check it inside framework boilerplate.

Reply:
but please check it